### PR TITLE
Create home screen

### LINKED ISSUE #39: feat/ui-prepare-home-screen
**AC**
Create homeScreen based on figma requirements

### PR DESCRIPTION
**Pull request by Mariusz-web**

### CODE REVIEW

**Comment by T0ffik** [comment]
clicking on register or sign in does not redirecting to sites

**Review by Veryyapeee — APPROVED** [review]
Imo looks cool (on mobile at least where I am checking)
Gj!

File: src/screens/Home/Home.tsx [hunk 1]
```diff
@@ -1,7 +1,59 @@
...
+                    TeamUp
+                </Typography>
+                <Fab color='primary' aria-label='logo' sx={style.fabLogo}>
+                    <GamepadRoundedIcon sx={{ fontSize: '48px' }} />
+                </Fab>
+            </Box>
+            <Typography variant='h6' color='white' sx={style.firstTypography}>
+                Don't wait anymore, find <br /> out your game mate now
+            </Typography>
+            <NavLink to={ROUTES.REGISTER} style={style.linkText}>
```
Comment (T0ffik) [inline src/screens/Home/Home.tsx:23]: I wonder if this navlink is necessary, we have button under so we can get rid of this tag and just add onclick to button, @Veryyapeee right?

Comment (Veryyapeee) [reply src/screens/Home/Home.tsx:23]: 1. Adding onClick is creating unnecessary  function and you have to get navigation object to run the method

2. Link component is semantic, just button is not in this case

Comment (Mariusz-web) [reply src/screens/Home/Home.tsx:23]: So now it is as you say :) if think so

**Review by maciejopalinski — APPROVED** [review]

**Pull request merged**